omod/src/main/webapp/resources/scripts/navigator/navigator.js

```diff
@@ -25,74 +25,82 @@ var Navigator = {isReady: false}
 
 
 // TODO figure out out to set up mocking to test this
+// TODO this really isn't just the "keyboard controller" anymore as it adds buttons and supports mouse clickers
 function KeyboardController(formElement) {
```

Comment:
I imagine we can't rename it because it's used in other modules?

Reply:
yeah, I believe so, didn't want to risk it at this point